src/pages/settings/bank-accounts/common/hooks/useBankAccountPages.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useTranslation } from 'react-i18next';
+
+export const useBankAccountPages = () => {
+  const { t } = useTranslation();
+
+  const pages = [
+    { name: t('settings'), href: '/settings' },
+    { name: t('bank_accounts'), href: '/settings/bank_accounts' },
+  ];
+
+  return pages;
+};
```

Comment:
There's no need for custom hook just to return breadcrumbs and it is anti-pattern to the app itself.

Reply:
> There's no need for custom hook just to return breadcrumbs and it is anti-pattern to the app itself.

Removed